### DISCUSSION TITLE
Add Shipping Carrier Parcll (cirro) to Etsy

### DISCUSSION THREAD

**Discussion by kiernangs** [etsy/open-api#1077] [Ideas]
I would like to request that etsy add Cirro to the list of trackable carriers on etsy.

Cirro ecommerce  is supported by aftership already so it should be easy to add.

Cirro: https://www.cirrotrack.com/

http://aftership.com/track?c=parcll

**Comment by kiernangs** [comment]
Because etsy already supports yunexpress , creating text to just auto recognize  Parcll as Yunexpress would also resolve this.

**Comment by emergingdzns** [comment]
@kiernangs Etsy staff doesn't monitor this board much, if at all. To make your request, you need to email developers@etsy.com. To get a new shipping carrier added, you'd have to provide an example tracking code and the url for directing the user to check the shipping status.

**Reply by kiernangs** [reply]
It would be lovely if that worked.  Sadly they love to ignore requests.

**Reply by Nikolasgrizli** [reply]
the same problem((

**Reply by kiernangs** [reply]
@Nikolasgrizli  sadly I dont expect etsy to put in any effort or care.

You can try emailing developers@etsy.com and see what they say.

**Reply by kiernangs** [reply]
> the same problem((

@Nikolasgrizli Turns out... after years of asking specifically if they had any form for requesting adding support for a new carrier (as recently as last week)  they did  have such a request form all along. They just didnt tell their support people.

I finally today talked to a support person named jack who sent me this etsy link for requesting carriers:  https://survey.alchemer.com/s3/7650632/Etsy-Feedback-Form-jaf267ho6w

**Comment by kiernangs** [comment]
Some day ...maybe  http://aftership.com/track?c=parcll

**Comment by kiernangs** [comment]
Turns out... after years of asking specifically if they had any form for requesting adding support for a new carrier (as recently as last week)  they did  have such a request form all along. They just didnt tell their support people.

I finally today talked to a support person named jack who sent me this etsy link for requesting carriers:  https://survey.alchemer.com/s3/7650632/Etsy-Feedback-Form-jaf267ho6w
